packages/gamut/src/AccordionButton/index.tsx

```diff
@@ -8,9 +8,21 @@ import ButtonBase from '../ButtonBase';
 
 export type AccordionButtonProps = {
   className?: string;
+  /**
+   * Whether or not the button should display as open or closed.
```

Comment:
Nit: _or not_ is filler.

```suggestion
   * Whether the button should display as open or closed.
```